cmd/nightly/forward-postgres-tester/helpers.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/uuid"
+
+	"sigs.k8s.io/yaml"
+)
+
+func (receiver applicationOpts) Validate() error {
```

Comment:
receiver -> o?

Reply:
Naw. "receiver" is the de facto standard. However, one usually shortens it to the first character of the struct's name